docs/16-using-cairo.md

```diff
@@ -75,7 +75,7 @@ of surfaces including:
  * Monochrome image surfaces with transparency (`A8`) or without (`A1`)
  * SVG vectorial surfaces
  * Native (XCB) surfaces
- * Framebuffers and other less interesting ones (from an Awesome's point of
+ * Framebuffers and other less interesting ones (from Awesome point of
    view)
```

Comment:
I agree with the removal of `an`, but I feel like `'s` should stay, no? (Same as above)